lighthouse-cli/printer.ts

```diff
@@ -19,7 +19,15 @@
 
 'use strict';
 
+/**
+ * An enumeration of acceptable output modes:
+ *   'pretty': Pretty print the results
+ *   'json': JSON formatted results
+ *   'html': An HTML report
+ */
+enum OutputMode { pretty, json, html };
```

Comment:
i think we should expose this enum in the places that need to call into checkOutputMode so it will just look like


`checkOutputMode(OutputMode.pretty)`

So there is no reason for the switch case to go from enum => string and string => enum

createOutput will still have the branching logic but we will only be operating on enum values which will be nice